src/components/Grid/GridRow.vue

```diff
@@ -0,0 +1,85 @@
+<template>
+  <tr>
+    <th>
+        <span class="label">
+          {{label}}
+        </span>
+    </th>
+    <td>
+        <facet-option class="selectRow"><font-awesome-icon icon="arrow-right" /></facet-option>
+    </td>
+    <td :key="index"
+        v-for="(count,index) in counts">
+          <facet-option
```

Comment:
i would suggest using a table to render the table instead of the facet option that was built to serve as a option for the facets

Reply:
is only being moved from the parent to the child